aws-cs-fargate/App/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  }
+}
```

Comment:
Nit, but I would omit this file

Reply:
Ok! Was autogenerated by `dotnet new`, wasn't sure what it is 😄